packages/account/src/Sections/Verification/ProofOfIdentity/idv.jsx

```diff
@@ -7,13 +7,13 @@ import { identity_status_codes } from './proof-of-identity-utils';
 
 const Idv = ({ handleRequireSubmission, idv, is_from_external, needs_poa, redirect_button }) => {
     const { status, submissions_left, last_rejected } = idv;
-
+    const is_report_flag_available = 'report_available' in idv;
```

Comment:
should we add'report_available' to a const file? 🤔

Reply:
i think its okay to be here. since if we move it a const file, we need to pass 'report_available' in idv as a prop, which doesnt make much sense